lib/active_fedora/loadable_from_json.rb

```diff
@@ -45,7 +45,11 @@ def to_s
       # It is expected that the singular filter gets applied after fetching the value from this
       # resource, so cast everything back to an array.
       def set_value(k, v)
-        @hash[k] = Array(v)
+        if (!@hash[k].nil? && @hash[k].is_a?(Array))
```

Comment:
Oh, I see it's appending vs replacing. Why would we append in a method called "set_value"?